Tutorial.md

```diff
@@ -0,0 +1,530 @@
+
+    ```
+    {
+        "typeName": "Testing::Unicorn::Maker",
+        "description": "An example resource that creates unicorns.",
+        "sourceUrl": "The URL of the source code for this resource, if public.",
+        "properties": {
+            "uid": {
+                "description": "The ID of the majestic animal",
+                "pattern": "^[a-z0-9]+$",
+                "type": "string"
+            },
+            "name": {
+                "description": "The name of the majestic animal",
+                "type": "string",
+                "pattern": "^[a-zA-Z0-9]+$",
+                "minLength": 3,
+                "maxLength": 250
+            },
+            "color": {
+                "description": "The color of the majestic animal",
+                "type": "string",
+                "pattern": "^[a-zA-Z0-9]+$",
+                "minLength": 3,
+                "maxLength": 250
+            }
+        },
+        "additionalProperties": false,
+        "required": [
+            "name",
+            "color"
+        ],
+        "readOnlyProperties": [
+            "/properties/uid"
+        ],
+        "primaryIdentifier": [
+            "/properties/uid"
+        ],
+        "handlers": {
+            "create": {
+                "permissions": []
+            },
+            "read": {
+                "permissions": []
+            },
+            "update": {
+                "permissions": []
+            },
+            "delete": {
+                "permissions": []
+            },
+            "list": {
+                "permissions": []
+            }
+        }
+    }
+    ```
+
+1. Update the auto\-generated files in the resource provider package so that they reflect the changes we've made to the resource provider schema\.
+
+   When we first initiated the resource provider project, the CloudFormation CLI generated supporting files and code for our resource provider\. Since we've made changes to the resource provider schema, we'll need to regenerate that code to ensure that it reflects the updated schema\. To do this, we use the generate command:
+
+   ```
+   $ cfn generate
+   Generated files for Testing::Unicorn::Maker
+   ```
+
+## Implement the Resource Handlers<a name="resource-type-walkthrough-implement"></a>
+
+Now that we have our resource provider schema specified, we can start implementing the behavior we want the resource provider to exhibit during each resource operation\. To do this, we'll have to implement the various event handlers, for Typescript all the handlers are in the same file `testing-unicorn-maker/src/handlers.ts`.
+
+## Helpers<a name="resource-type-walkthrough-implement-helpers"></a>
+
+At the top of the file below the existing imports add:
+
+```
+import fetch, { Response } from 'node-fetch';
+
+// Use this logger to forward log messages to CloudWatch Logs.
```

Comment:
With the new support lib version, the logger proxy is being passed to the handler instead (similar to the java one).

Reply:
i have created the following PR containing the updated logger: https://github.com/brianterry/unicorn-maker/pull/10